### PR TITLE
[salus] Add `running-state` channel for it600

### PR DESCRIPTION
**Pull request by magx2**
# Add `work_type` channel for it600

- [salus] [it600] Add `running-state` for it600 devices

### CODE REVIEW

File: bundles/org.openhab.binding.salus/src/main/resources/OH-INF/update/it600-running-state.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="salus:salus-it600-device">
+		<instruction-set targetVersion="4">
```
Comment (lsiepel) [inline bundles/org.openhab.binding.salus/src/main/resources/OH-INF/update/it600-running-state.xml:7]:
```suggestion
		<instruction-set targetVersion="1">
```

This has to match the `thingTypeVersion` thjat is set on the thing xml

File: bundles/org.openhab.binding.salus/src/main/resources/OH-INF/thing/it600.xml [hunk 2]
```diff
@@ -21,7 +21,11 @@
 			<channel id="temperature" typeId="it600-temp-channel"/>
 			<channel id="expected-temperature" typeId="it600-expected-temp-channel"/>
 			<channel id="work-type" typeId="it600-work-type-channel"/>
+			<!-- `running-state` added in `update/it600-running-state.xml` -->
```
Comment (lsiepel) [inline bundles/org.openhab.binding.salus/src/main/resources/OH-INF/thing/it600.xml:24]: The newly added channel should be listed here.
```suggestion
			<channel id="running-state" typeId="it600-running-state"/>
```

File: bundles/org.openhab.binding.salus/src/main/resources/OH-INF/i18n/salus.properties [hunk 3]
```diff
@@ -22,6 +22,7 @@ thing-type.config.salus.salus-aws-bridge.clientId.description = The app client I
 thing-type.config.salus.salus-aws-bridge.companyCode.label = Company Code
 thing-type.config.salus.salus-aws-bridge.group.aws.label = AWS
 thing-type.config.salus.salus-aws-bridge.group.aws.description = AWS Properties
+thing-type.config.salus.salus-aws-bridge.identityPoolId.label = Identity Pool ID
```
Comment (lsiepel) [inline bundles/org.openhab.binding.salus/src/main/resources/OH-INF/i18n/salus.properties:25]: This label was missing before? Does not seem to be related to this PR.

**Review by lsiepel — CHANGES_REQUESTED** [review]
Thanks! Besides these comments, it would also be nice if you can update the readme.md with the added channel.

File: bundles/org.openhab.binding.salus/src/main/resources/OH-INF/i18n/salus.properties (hunk 3 above)
Comment (magx2) [reply bundles/org.openhab.binding.salus/src/main/resources/OH-INF/i18n/salus.properties:25]: Ive just run i8n for the plugin and this poped up

**Comment by magx2** [comment]
@lsiepel just to be sure after your 2 suggestions: I had a use case that when the update changelog could not run cause the channel already existed. I don't remember how i came up with this, but I did it somehow on my local/debug env

File: bundles/org.openhab.binding.salus/README.md [hunk 4]
```diff
@@ -75,11 +75,12 @@ removed.
...
+| Channel                    | Type               | Read/Write | Description                                                                                                                                                                                                                                                                                     |
+|----------------------------|--------------------|------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| it600-temp-channel         | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.salus/README.md:80]: According to the it600.xml this channel id is different. Looks like you added the type id, while it should bne the channel id here:
```suggestion
| temperature         | Number:Temperature | RO         | Current temperature in the room                                                                                                                                                                                                                                                                 |
```

File: bundles/org.openhab.binding.salus/README.md [hunk 5]
```diff
@@ -75,11 +75,12 @@ removed.
...
 | it600-expected-temp-channel | Number:Temperature | RW         | Sets the desired temperature in the room                                                                                                                                                                                                                                                        |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.salus/README.md:81]:
```suggestion
| expected-temperature | Number:Temperature | RW         | Sets the desired temperature in the room                                                                                                                                                                                                                                                        |
```

According to the it600.xml this channel id is different. Looks like you added the type id, while it should be the channel id here.

File: bundles/org.openhab.binding.salus/README.md [hunk 6]
```diff
@@ -75,11 +75,12 @@ removed.
...
-| it600-work-type-channel     | String             | RW         | Sets the work type for the device. OFF - device is turned off MANUAL - schedules are turned off, following a manual temperature set, AUTOMATIC - schedules are turned on, following schedule, TEMPORARY_MANUAL - schedules are turned on, following manual temperature until the next schedule. |
+| it600-work-type-channel    | String             | RW         | Sets the work type for the device. OFF - device is turned off MANUAL - schedules are turned off, following a manual temperature set, AUTOMATIC - schedules are turned on, following schedule, TEMPORARY_MANUAL - schedules are turned on, following manual temperature until the next schedule. |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.salus/README.md:82]:
```suggestion
| work-type | String             | RW         | Sets the work type for the device. OFF - device is turned off MANUAL - schedules are turned off, following a manual temperature set, AUTOMATIC - schedules are turned on, following schedule, TEMPORARY_MANUAL - schedules are turned on, following manual temperature until the next schedule. |
```

According to the it600.xml this channel id is different. Looks like you added the type id, while it should be the channel id here.

File: bundles/org.openhab.binding.salus/README.md [hunk 7]
```diff
@@ -75,11 +75,12 @@ removed.
...
+| it600-running-state     | Switch             | RO         | Is the device running |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.salus/README.md:83]:
```suggestion
| running-state     | Switch             | RO         | Is the device running |
```
According to the it600.xml this channel id is different. Looks like you added the type id, while it should be the channel id here.

**Review by lsiepel — CHANGES_REQUESTED** [review]
Only now noticed the docs where wrong. Upgrade instructions are fine now, thanks!

**Review by lsiepel — APPROVED** [review]
